src/inc/common/Mlp_Helpers.php

```diff
@@ -478,8 +478,16 @@ public static function show_linked_elements( $args ) {
 		}
 
 		$output .= '</ul></div>';
-
-		return apply_filters( 'show_linked_elements', $output, $items, $params );
+		
+		/**
+		 * Filter the output of the linked elementes
+		 * 
+		 * @param	string $output the generated HTML
+		 * @param	array $items the language items
+		 * @param	array $params the parameters which came from the original call
+		 */
+		$output = apply_filters( 'mlp_linked_elements_html', $output, $items, $params );
```

Comment:
Well, [I explicitly asked for this](https://github.com/inpsyde/multilingual-press/pull/228#discussion_r76737373). ;)